ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/advanced-gas-fee-inputs-basefee.js

```diff
@@ -0,0 +1,158 @@
+import React, { useCallback, useState } from 'react';
+import { useSelector } from 'react-redux';
+
+import { PRIORITY_LEVELS } from '../../../../../shared/constants/gas';
+import {
+  divideCurrencies,
+  multiplyCurrencies,
+} from '../../../../../shared/modules/conversion.utils';
+import { PRIMARY, SECONDARY } from '../../../../helpers/constants/common';
+import {
+  decGWEIToHexWEI,
+  hexWEIToDecGWEI,
+} from '../../../../helpers/utils/conversions.util';
+import { useGasFeeContext } from '../../../../contexts/gasFee';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { useUserPreferencedCurrency } from '../../../../hooks/useUserPreferencedCurrency';
+import { useCurrencyDisplay } from '../../../../hooks/useCurrencyDisplay';
+import Button from '../../../ui/button';
+import FormField from '../../../ui/form-field';
+import I18nValue from '../../../ui/i18n-value';
+
+import AdvancedGasFeeInputSubtext from '../advanced-gas-fee-input-subtext';
+import { getAdvancedGasFeeValues } from '../../../../selectors';
+
+const devideCurrencyValues = (value, baseFee, numberOfDecimals) => {
+  if (baseFee > 1)
+    return divideCurrencies(decGWEIToHexWEI(value), baseFee, {
+      numberOfDecimals,
+      dividendBase: 16,
+      divisorBase: 16,
+      toNumericBase: 'dec',
+    });
+  return 1;
+};
+
+const multiplyCurrencyValues = (baseFee, value, numberOfDecimals) =>
+  multiplyCurrencies(baseFee, value, {
+    numberOfDecimals,
+    multiplicandBase: 16,
+    multiplierBase: 10,
+    fromNumericBase: 'hex',
+    toNumericBase: 'dec',
+    fromDenomination: 'WEI',
+    toDenomination: 'GWEI',
+  });
+
+const AdvancedGasFeeInputBaseFee = () => {
```

Comment:
What do you think about renaming this to MaxBaseFeeInput?

Reply:
👍 changed